style.css

```diff
@@ -630,3 +647,430 @@
+    color: #344563;
+    margin-left: 1.2rem;
+  }
+
+  .flex .flex-box p {
+    font-size: 16px;
+    font-family: var(--font-family);
+    color: #344563;
+  }
+
+  #first-sec p {
+    font-size: var(--font-size-p-desktop);
+    font-family: var(--font-family);
+    text-align: justify;
+    padding-top: -1rem;
+    margin-top: -1rem;
+  }
+
+  .flex .flex-box h3 {
+    font-family: var(--font-family);
+    font-size: 16px;
+    color: #7f8cff;
+  }
+
+  #first-sec h3 {
+    font-family: var(--font-family);
+    font-size: var(--font-size-h3-desktop);
+    color: #7f8cff;
+    text-align: justify;
+    padding-top: -1rem;
+    margin-top: -0.5rem;
+  }
+
+  .contact-form form ul {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem 10vw 1rem 10vw;
+  }
+
+  #first-sec ul {
+    display: flex;
+    margin-left: -3.5rem;
+    padding-top: -1rem;
+    margin-top: -1rem;
+  }
+
+  .div-info .sec-list li {
+    float: left;
+    margin: 1vw;
+    background-color: #ebebff;
+    padding: 0.5rem 1rem 0.5rem 1rem;
+    border-radius: 8px;
+    height: 2vw;
+  }
+
+  .flex .flex-box .social-img li {
+    float: left;
+    margin: 1vw;
+  }
+
+  .flex-3 .grid .grid-2 .language li {
+    float: left;
+    background-color: #f7f7f9;
+    margin: 1vw;
+    font-size: 15px;
+    color: #253858;
+    width: 9vw;
+    height: 9vw;
+    border-radius: 4px;
+  }
+
+  #first-sec ul li {
+    margin: 0.4rem 1rem 0 1rem;
+  }
+
+  .work-sec {
+    display: none;
+  }
+
+  .desktop-work-sec {
+    display: grid;
+    gap: 3.5rem;
+    grid-template-columns: 1;
+    grid-template-rows: repeat(4, 40rem);
+    height: 90rem;
+    margin: 5rem 6vw 50vh 6vw;
+  }
+
+  .desk-first-div,
+  .desk-second-div,
+  .desk-third-div,
+  .desk-fourth-div,
+  .desktop-work-sec,
+  .desk-div-img img {
+    grid-column: 1;
+    width: 40vw;
+    height: 30vw;
+  }
+
+  .desktop-work-sec .desk-first-div {
+    grid-column: 1;
+    grid-row: 1;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 2vw;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-second-div {
+    grid-column: 1;
+    grid-row: 2;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 2vw;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-third-div {
+    grid-column: 1;
+    grid-row: 3;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 20px;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
+
+  .desktop-work-sec .desk-fourth-div {
+    grid-column: 1;
+    grid-row: 4;
+    background-color: #fff;
+    display: grid;
+    grid-template-columns: repeat(2, 40vw);
+    gap: 1rem;
+    padding: 20px;
+    width: 83vw;
+    height: 24rem;
+    border-radius: 16px;
+  }
```

Comment:
- [ ] The image 📷  of the `Uber Navigation` project card is on the right side ➡️  in [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z0umLtHjW2jvzRBK-0) please ☺️  make sure yours is also placed on the right. ➡️ 

**_Figma_**

<img width="692" alt="Screenshot 2023-02-23 at 9 29 57 PM" src="https://user-images.githubusercontent.com/44798044/220978659-ad7ed668-1d6c-4be9-a0e4-3bd34a568a6d.png">

**_Yours_**

<img width="947" alt="Screenshot 2023-02-23 at 9 30 24 PM" src="https://user-images.githubusercontent.com/44798044/220979316-85af5c1b-6fc1-45e8-87e3-f4dc967dfc0a.png">